sdk/core/azure-core-amqp/samples/eventhub_reader_sample/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright(c) Microsoft Corporation.All rights reserved.
+# SPDX - License - Identifier : MIT
+
+cmake_minimum_required(VERSION 3.15)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Why do we need this?

Reply:
I have absolutely no idea why we need this. Cargo Cult programming.